junit-jupiter-api/src/main/java/org/junit/jupiter/api/MethodOrderer.java

```diff
@@ -172,6 +175,76 @@ private static int getOrder(MethodDescriptor descriptor) {
 		}
 	}
 
+	/**
+	 * {@code MethodOrderer} that sorts methods based on the {@link Depend @Depend}
+	 * annotation.
+	 */
+	class DependAnnotation implements MethodOrderer {
+		private static final Logger logger = LoggerFactory.getLogger(Random.class);
+
+		private static Map<String, Integer> dependencySize;
+		private static Map<String, String[]> digraph;
```

Comment:
What is the meaning of `digraph` in this context?

Please either rename it or document its purpose.

Reply:
I have remove all states in the class and provide these as parameters to depthFirstSearch. I have also added an explanation for what it is used for in comment.